ethereum/eip712-v2.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "properties": {
+        "blockchainName": {
+            "enum": [
+                "ethereum"
+            ]
+        },
+        "chainId": {
+            "enum": [
+                1
+            ]
+        },
+        "contracts": {
+            "items": {
+                "properties": {
+                    "address": {
+                        "pattern": "^0x[a-z0-9]{40}$",
+                        "type": "string"
+                    },
+                    "contractName": {
+                        "type": "string"
+                    },
+                    "messages": {
+                        "items": {
+                            "additionalProperties": false,
+                            "properties": {
+                                "mapper": {
+                                    "additionalProperties": false,
+                                    "properties": {
+                                        "fields": {
+                                            "items": {
+                                                "additionalProperties": false,
+                                                "properties": {
+                                                    "assetPath": {
+                                                        "type": "string"
+                                                    },
+                                                    "format": {
+                                                        "enum": [
```

Comment:
We also need a `raw` value for unformatted fields, see https://github.com/LedgerHQ/app-ethereum/blob/feat/apa/eip712_improvements/doc/ethapp.adoc#show-raw-field

Reply:
yes, I missed it since I relied on [https://ledgerhq.atlassian.net/wiki/spaces/PTX/pages/4404903999/ARCHI+EIP-712+improvements](https://ledgerhq.atlassian.net/wiki/spaces/PTX/pages/4404903999/ARCHI+EIP-712+improvements) which does not match the code you show me.